runtime/near-vm-runner/src/cache.rs

```diff
@@ -99,6 +99,16 @@ pub trait ContractRuntimeCache: Send + Sync {
     fn has(&self, key: &CryptoHash) -> std::io::Result<bool> {
         self.get(key).map(|entry| entry.is_some())
     }
+    /// TESTING ONLY: Clears the cache including in-memory and persistent data (if any).
+    ///
+    /// This should be used only for testing, since the implementations may not provide
+    /// a consistent view when the cache is both cleared and accessed as the same time.
+    ///
+    /// Default implementation panics; the implementations for which this method is called
+    /// should provide a proper implementation.
+    fn test_only_clear(&self) -> std::io::Result<()> {
```

Comment:
Is it possible to hide this function behind a compilation flag set only in integration tests? Assuming we have such feature

Reply:
`#[cfg(test)]` would work for unittests, but it does not work for integration tests, so we would need a new feature for that. I guess it could be guarded with `test_features`.

Reply:
`test_features` is actually meant for adversarial features ([related thread](https://near.zulipchat.com/#narrow/stream/295302-general/topic/.E2.9C.94.20Adding.20.22test_features.22.20to.20CI.20tests/near/446039809)) but I think clearing cache kind of fits into that category.

Reply:
conditioned the code with `test_features`, as it fits in the purpose of this flag.